components/brave_new_tab_ui/async/today.ts

```diff
@@ -4,40 +4,38 @@
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
 import AsyncActionHandler from '../../common/AsyncActionHandler'
-import * as Background from '../../common/Background'
 import * as Actions from '../actions/today_actions'
 import { ApplicationState } from '../reducers'
 import { saveIsBraveTodayOptedIn } from '../api/preferences'
+import getBraveNewsController, * as BraveNews from '../api/brave_news'
 
 function storeInHistoryState (data: Object) {
   const oldHistoryState = (typeof history.state === 'object') ? history.state : {}
   const newHistoryState = { ...oldHistoryState, ...data }
   history.pushState(newHistoryState, document.title)
 }
 
-import Messages = BraveToday.Messages
-import MessageTypes = Background.MessageTypes.Today
-
 const handler = new AsyncActionHandler()
 
 handler.on(Actions.todayInit.getType(), async (store, payload) => {
   // Let backend know that a UI with today is open, so that it can
   // pre-fetch the feed if it is not already in a cache.
-  await Background.send(MessageTypes.indicatingOpen)
+  // await Background.send(MessageTypes.indicatingOpen)
```

Comment:
we don't need todayInit action anymote?

Reply:
I think that this is unneccessary as it will only affect the first NTP page that the user scrolls down to NTP. They will briefly see a loading screen which is ok. And there are timers to update the data periodically. And now I just added a "prefetch" timer in BraveNewsController that will ensure there is data 1 minute after class is created or preference is switched on..